src/overlays/actors/ovl_En_Talk/z_en_talk.h

```diff
@@ -14,4 +14,6 @@ typedef struct EnTalk {
 
 extern const ActorInit En_Talk_InitVars;
 
+#define EN_TALK_GET_TEXT_ID(this) ((this->actor.params & 0x3F) + 0x1C00)
```

Comment:
```suggestion
#define EN_TALK_GET_TEXT_ID(thisx) (((thisx)->params & 0x3F) + 0x1C00)
```
Our current convention is to use the base actor pointer with params macros.